.github/workflows/ibis-backends.yml

```diff
@@ -680,6 +680,9 @@ jobs:
             deps:
               - setuptools@75.1.0
               - delta-spark@3.2.1
+              - googleapis-common-protos
+              - grpcio
+              - grpcio-status
```

Comment:
This turned out to be required. I'm not sure how this was passing before except that some other dependency was pulling in these `connect`-required dependencies.